sdk/src/auth/index.ts

```diff
@@ -367,11 +367,20 @@ class AWF_Auth implements AWF_IAuth {
     await this.hubUser.setupMailbox();
     const mailboxID = await this.hubUser.getMailboxID();
     this.inboxWatcher = await this.hubUser.watchInbox(mailboxID, ev => {
-      if (ev?.message?.body && ev?.message?.readAt === 0) {
+      if (
+        ev?.message?.body &&
```

Comment:
this can be simplified
if(ev?.message?.body && ev?.message?.readAt === 0){
  if(ev?.message?.from === process.env.EWA_MAILSENDER){ // emit new_notif}
  else{ // emit new message}
}